gslib/tests/test_label.py

```diff
@@ -284,10 +284,7 @@ def testChMinusDWorksWithoutExistingLabels(self):
     self.RunGsUtil(['label', 'ch', '-d', 'dummy-key', suri(bucket_uri)])
     stdout = self.RunGsUtil(['label', 'get', suri(bucket_uri)],
                             return_stdout=True)
-    if self._use_gcloud_storage:
-      self.assertIn('[]', stdout)
-    else:
-      self.assertIn('%s/ has no label configuration.' % suri(bucket_uri),
+    self.assertIn('%s/ has no label configuration.' % suri(bucket_uri),
```

Comment:
Because Now we have gsutiljson formatter which basically prints similar output as gsutil for some of the commands. No need to check for different output of shimmed gsutil command.

Reply:
Yes!